<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="img_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>701</width>
      <height>571</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string notr="true">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>231</width>
      <height>591</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="Editing">
     <attribute name="title">
      <string>Editing</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="textWb">
          <property name="text">
           <string>WhiteBalance:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QSpinBox" name="green_spin">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="red_spin">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QSlider" name="contrast_slider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="textGreen">
          <property name="text">
           <string>Green</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="textRed">
          <property name="text">
           <string>Red</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSlider" name="red_slider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSlider" name="green_slider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QSpinBox" name="contrast_spin">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="textBlue">
          <property name="text">
           <string>Blue</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSlider" name="blue_slider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="12" column="1" colspan="2">
         <widget class="QPushButton" name="HSL_btn">
          <property name="text">
           <string>HSL</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSlider" name="exposure_slider">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="textContrast">
          <property name="text">
           <string>Contrast</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QSpinBox" name="eposure_spin">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="1">
         <widget class="QLabel" name="textExposure">
          <property name="text">
           <string>Exposure</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1" colspan="2">
         <widget class="QPushButton" name="reset_btn">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" rowspan="2">
         <widget class="QSpinBox" name="blue_spin">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Kernel">
     <attribute name="title">
      <string>Kernel</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Gaussian</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Sharpener</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="gaussian_slider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="sepia_btn">
         <property name="text">
          <string>Sepia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grayscale_btn">
         <property name="text">
          <string>GrayScale</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="geometry_btn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>340</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Geometry</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Paint">
     <attribute name="title">
      <string>Paint</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="open_btn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_btn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="new_btn">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="textZoom">
       <property name="text">
        <string>Zoom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="zoom_slider">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuZeroPane">
    <property name="title">
     <string>ZeroPane</string>
    </property>
   </widget>
   <addaction name="menuZeroPane"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>red_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>883</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>947</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exposure_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>eposure_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>951</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_spin</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>943</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>red_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>red_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>944</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>862</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>green_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>green_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>946</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>871</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>blue_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>blue_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>939</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>851</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eposure_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>exposure_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>940</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>878</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrast_spin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>contrast_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>936</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
